gpytorch/variational/grid_interpolation_variational_strategy.py

```diff
@@ -30,6 +30,11 @@ class GridInterpolationVariationalStrategy(_VariationalStrategy):
     :param list grid_bounds: Bounds of each dimension of the grid (should be a list of (float, float) tuples)
     :param ~gpytorch.variational.VariationalDistribution variational_distribution: A
         VariationalDistribution object that represents the form of the variational distribution :math:`q(\mathbf u)`
+
+    :ivar grid: The grid of points that the inducing points are based on.
+        The grid is stored as a matrix, where each column corresponds to the
+        projection of the grid onto one dimension.
+    :type grid: torch.Tensor (M x D)
     """
 
     def __init__(self, model, grid_size, grid_bounds, variational_distribution):
```

Comment:
We could add type hints to this, while we are at it, but I'll leave that up to you.